.github/workflows/rebase.yml

```
@@ -17,6 +18,10 @@ on:
         description: 'Rebase branch'
         required: true
         default: 'rebase-upstream'
+      backup_branch:
+        description: 'Backup branch'
+        required: true
+        default: 'rebase-upstream-backup'
```

```suggestion
      backup_branch:
        description: 'Head of origin branch from the point of time when rebase started'
        required: true
        default: 'rebase-upstream-backup'
```

Tried to come up with a more descriptive name than "backup branch" but was not able to come up with something not long, so maybe just make the description more descriptive?

---

Descriptions improved.